etc/cms-resource-reporter.yml

```diff
@@ -0,0 +1,120 @@
+# etc/cms-resource-reporter.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Handles CMS Resource SNS Events
+Parameters:
+  SlackMessageRelayTopicArn:
+    Type: String
+  CmsSeriesCreateSnsTopicArn:
+    Type: String
+Resources:
+  LambdaIamRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+            - Effect: "Allow"
+              Principal:
+                Service:
+                  - "lambda.amazonaws.com"
+              Action:
+                - "sts:AssumeRole"
+      Path: "/"
+      Policies:
+        - PolicyName: SnsPolicy
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "sns:Publish"
+                Resource:
+                  - !Ref SlackMessageRelayTopicArn
+      ManagedPolicyArns:
+        - "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
+  PushSubscription:
+    Type: "AWS::SNS::Subscription"
+    Properties:
+      Endpoint: !GetAtt MessageHandlerFunction.Arn
+      Protocol: "lambda"
+      Region: "us-east-1"
+      TopicArn: !Ref CmsSeriesCreateSnsTopicArn
+  LambdaResourcePolicy:
+    Type: "AWS::Lambda::Permission"
+    Properties:
+      FunctionName: !GetAtt MessageHandlerFunction.Arn
+      Principal: "sns.amazonaws.com"
+      Action: "lambda:InvokeFunction"
+      SourceArn: !Ref CmsSeriesCreateSnsTopicArn 
+  MessageHandlerFunction:
+    Type: "AWS::Lambda::Function"
+    Properties:
+      Code:
+        ZipFile: >
+          var AWS = require("aws-sdk");
+          exports.handler = async (event, context) => {
+              var message = JSON.parse(event["Records"][0]["Sns"]["Message"]);
+              var event = JSON.parse(message["body"]);
+
+              var resourceLink = event['_links']['self']['href'];
+              var fromImport = event['_links']['prx:podcast-imports']['count'] > 0;
+
+
+              var importLabel = fromImport ? "import" : "publish"
+              var eventText = `A new series was created on CMS via *${importLabel}*:
+          _${event['title']}_
```

Comment:
I like the accurate labeling.  Is this multiline thing necessary to make the slack formatting look nice?